webpack.config.common.js

```diff
@@ -81,6 +81,10 @@ module.exports = {
         },
         test: /\.svg$/,
       },
+      {
+        loader: 'raw-loader',
+        test: /\.glsl$/,
+      },
```

Comment:
We need the raw loader in order to load the glsl files as text into our js bundle